src/Microsoft.AspNet.OData.Shared/Interfaces/IWebApiContext.cs

```diff
@@ -35,6 +36,11 @@ internal interface IWebApiContext
         /// </summary>
         Uri DeltaLink { get; set; }
 
+        /// <summary>
+        /// Page size to be used by skiptoken implementation for the top-level resource for the request. 
+        /// </summary>
+        int PageSize { get; set; }
```

Comment:
>int [](start = 8, length = 3)

maybe int?

Reply:
Since the underlying public methods that consume this take in an int, it will be a breaking change there. Otherwise, I would have to additionally convert null values to zero before calling those APIs. 

---
In reply to: [281406208](https://github.com/OData/WebApi/pull/1724#discussion_r281406208) [](ancestors = 281406208)